bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/dao/impl/PatientDaoImpl.java

```diff
@@ -142,15 +160,14 @@ public PatientDaoImpl(SessionFactory sessionFactory) {
         return identifierTypeNames;
     }
 
-    private void addIdentifierTypeName(List<String> identifierTypeNames,String identifierProperty) {
+    private void addIdentifierTypeName(List<String> identifierTypeNames, String identifierProperty) {
         String identifierTypes = Context.getAdministrationService().getGlobalProperty(identifierProperty);
         if(StringUtils.isNotEmpty(identifierTypes)) {
             String[] identifierUuids = identifierTypes.split(",");
-            for (String identifierUuid :
-                    identifierUuids) {
+            for (String identifierUuid : identifierUuids) {
                 PatientIdentifierType patientIdentifierType = Context.getPatientService().getPatientIdentifierTypeByUuid(identifierUuid);
                 if (patientIdentifierType != null) {
-                    identifierTypeNames.add(patientIdentifierType.getName());
+                    identifierTypeNames.add(i18n.getMessageKey(patientIdentifierType.getName()));
```

Comment:
@angshu this is basically the only other place where `Internationalizer` is used. Also altogether the only code location where it this interface is injected (and used) is in `PatientDaoImpl`.